src/main/java/com/blackducksoftware/integration/hub/jenkins/PostBuildHubScan.java

```diff
@@ -199,6 +202,15 @@ public boolean perform(final AbstractBuild<?, ?> build, final Launcher launcher,
         return true;
     }
 
+    private boolean isFailureConditionsConfigured(final AbstractBuild<?, ?> build) {
+        for (final Publisher publisher : ((FreeStyleProject) build.getParent()).getPublishersList()) {
+            if (publisher instanceof HubFailureConditionStep) {
```

Comment:
This should be 
`if (publisher instanceof HubFailureConditionStep) {
 +                return true;
 +            }`